code/build.gradle

```diff
@@ -38,7 +38,7 @@ ext {
 
     // dependencies
     junitVersion = "1.1.3"
-    mockitoCoreVersion = "2.23.0"
+    mockitoCoreVersion = "4.5.1"
```

Comment:
nit: I was told that the standardized name is `mockitoVersion` now.

Reply:
😂 yo, this came back to me as a 🪃